index.js

```diff
@@ -38,6 +38,51 @@ searchBtnCmp.addEventListener("click", function() {
             tempUnitCmp.innerHTML = "&#8451";
             newWindVal.innerHTML = Math.round(data.wind.speed);
             weatherWindCmp.appendChild(newWindVal);
+            let weatherIcon = data.weather[0].icon;
+            let icons = new Skycons({ "color": "white" });
```

Comment:
It's not working because op copy-paste problem. 
If you look at the example from the website it says:
```js
var skycons = new Skycons({"color": "pink"});
...
skycons.add("icon1", Skycons.PARTLY_CLOUDY_DAY);
skycons.play();
```
but you do
```js
let icons = new Skycons({ "color": "white" });
...
skycons.add(document.getElementById("weatherIcon"), Skycons.FOG);
skycons.play();
```

Reply:
Don't forget the debugging methods we've showed to you. You always can use them to identify where the problem is.